tests/test_activate.py

```diff
@@ -1976,18 +1978,17 @@ def get_env_var(self, env_var, default=None):
             self.sendline("@echo %%%s%%" % env_var)
             self.expect("@echo %%%s%%\r\n([^\r]*)\r" % env_var)
             value = self.p.match.groups()[0]
-        elif self.shell_name == 'powershell':
+        elif self.shell_name in ('powershell', 'pwsh'):
```

Comment:
it is called pwsh on mac and IDK on windows, but, the last else: sortof-worked for powershell